MoneyDataType/KnownCurrencyTable.cs

```diff
@@ -35,7 +35,12 @@ private static void InitCurrencyCodeTable()
                     .Distinct(new CurrencyEqualityComparer())
                     .ToDictionary(k => k.CurrencyIsoCode, e => e);
 
-                CurrencyTable.Add("BTC", new Currency("BitCoin", "₿", "BTC", 8));
+                // Make sure the CurrentCulture currency is used for countres with multiple cultures/languages.
+                var currentCultureCurrency = new Currency(CultureInfo.CurrentCulture) as ICurrency;
+                CurrencyTable.Remove(currentCultureCurrency.CurrencyIsoCode);
+                CurrencyTable.Add(currentCultureCurrency.CurrencyIsoCode, currentCultureCurrency);
```

Comment:
I still don't understand how this works. Why would you replace this in a static method? You can't know that what is the current culture when this runs (which is pretty much non-deterministic) is going to remain the current culture. The current culture potentially changes all the time. The whole idea of defaulting to the current culture sounds good on the surface, but I don't think it holds water. The default culture of the site may be a better choice, but even if you do that, you would certainly not modify the table (especially not in a static method: all tenants share that if I'm not mistaken), you'd instead select the correct current culture when it's queried for.

Reply:
NativeName is currently never shown so this change is rolled back.